website/client/js/controllers/filtersCtrl.js

```diff
@@ -17,6 +17,7 @@ habitrpg.controller("FiltersCtrl", ['$scope', '$rootScope', 'User', 'Shared',
             User.updateTag({params:{id:tag.id}, body:{name:tag.name}});
         })
         $scope._editing = false;
+        User.sync();
```

Comment:
I might have to look at it more closely, but you should just be able to remove it from the array.